### ISSUE TITLE
Improve console output of config

### ISSUE THREAD

**Issue by dialexo** [graasp/graasp-websockets#13]
Improve the console print of the boot message:

https://github.com/graasp/graasp-websockets/blob/9757a6a57848138c922c9dd9843d0cc611e1f7b8/src/service-api.ts#L398

Currently outputs:
```
{"level":30,"time":1625759719616,"pid":26801,"hostname":"yuudachi-zor","msg":"graasp-websockets: plugin booted with prefix /ws and Redis config {\"config\":{\"port\":6379,\"host\":\"127.0.0.1\"},\"notifChannel\":\"graasp-notif\"}"}
```

Could use `util.inspect()`: https://stackoverflow.com/questions/10729276/how-can-i-get-the-full-object-in-node-jss-console-log-rather-than-object

**Comment by juancarlosfarah** [comment]
@dialexo, also, do not log the password.

**Comment by dialexo** [comment]
Implemented in 0bad934580cbc6fb3f1f467a91aa1b13d99e5346
